crates/swc_common/src/errors/emitter.rs

```diff
@@ -41,6 +43,8 @@ pub trait Emitter: crate::sync::Send {
     fn should_show_explain(&self) -> bool {
         true
     }
+
+    fn emit_diagnostics(&mut self, _de: &RefCell<dyn DiagnosticEmitter>) {}
```

Comment:
`RefCell` this trait is too much assumption about the trait and `DiagnosticEmitter`